src/components/ranking/Ranking.tsx

```diff
@@ -0,0 +1,143 @@
+import { PaddingMd } from "../padding/Padding";
+import styled from "@emotion/styled";
+import RankingItem from "./RankingItem";
+import { PaddingLg } from './../padding/Padding';
+const mockRankingProducts = {
+  id: 123,
+  name: "BBQ 양념치킨+크림치즈볼+콜라1.25L",
+  imageURL:
+    "https://st.kakaocdn.net/product/gift/product/20231030175450_53e90ee9708f45ffa45b3f7b4bc01c7c.jpg",
+  price: {
+    basicPrice: 29000,
+    discountRate: 0,
+    sellingPrice: 29000,
```

Comment:
소소하게 큰 숫자를 사용할 땐 가독성을 위해 numeric separator를 사용해보시면 좋을거 같아요
```suggestion
    sellingPrice: 29_000,
```